openapi/external.yaml

```diff
@@ -35,7 +36,129 @@ paths:
           description: See Others
         "400":
           description: Validation Failed
-        "404":
-          description: Not Found
+  /api/v2/validate-profile-email:
+    get:
+      operationId: getTokenInfo
+      description: |
+        Validates the token and get email relative to a token.
+      parameters:
+        - in: query
+          name: token
+          required: true
+          schema:
+            $ref: "#/components/schemas/ValidationToken"
+      responses:
+        "200":
+          description: Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetTokenInfoResponse'
+        "400":
+          description: Validation failed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+        "500":
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+    post:
+      operationId: validateProfileEmail
+      description: |
+        Validates the token and updates the user profile.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ValidateProfileEmailPayload"
+      responses:
+        "204":
```

Comment:
All the status like for the `get` should be returned in this api

Reply:
changed in https://github.com/pagopa/io-functions-public/pull/76/commits/317e91956f0411a4d23dc3131b3cae2bbcb7caff